src/sql.rs

```diff
@@ -630,7 +631,18 @@ async fn maybe_add_from_param(
         .query_map(
             query,
             paramsv![],
-            |row| row.get::<_, String>(0),
+            // cs: maybe the following could be done even more simple than here?
+            |row| {
+                let value = match row.get_raw(0) {
+                    ValueRef::Text(t) => String::from_utf8_lossy(t).to_string(),
+                    _ => {
+                        let s_err = "maybe_add_from_param() - error - get_raw(0)".to_string();
+                        warn!(context, "{}", s_err);
+                        s_err
```

Comment:
Why are you returning the error msg here?

Reply:
please see utf8-fix. This is accidently into this pr, but not wrong.

Reply:
reset this to master file